modules/server/src/network/gql.ts

```diff
@@ -0,0 +1,18 @@
+import { print } from 'graphql';
+
+export const createGqlQuery = ({ query: queryAST, variables }) => {
+	// queries need to be strings when using regular http
+	const query = print(queryAST);
+	return JSON.stringify({ query, variables });
+};
+
+export const fetchGql = ({ url, gqlRequest }) => {
+	const query = JSON.stringify(gqlRequest);
+	const options = {
+		method: 'POST',
+		headers: { 'Content-Type': 'application/json' },
+		body: query,
+	};
+
+	return fetch(url, options);
```

Comment:
I'd prefer we used Axios, to keep things consistent with the rest of the server (unless I'm mistaken, which is always possible/likely).

Open to discuss, though!

Reply:
makes sense! easy one